config/templates/pmlogger/pmlogger-supremm.config

```diff
@@ -227,6 +227,9 @@ log mandatory on %{standard_freq} {
 	hotproc.psinfo.ppid
 	hotproc.psinfo.cmd
 	hotproc.psinfo.pid
+        hotproc.psinfo.processor
```

Comment:
I dislike the mixing of indentation types... but not my code so I will ignore ...

Reply:
Yeah, that file had some issues.  Should be fixed now.